src/hotspot/cpu/zero/frame_zero.inline.hpp

```diff
@@ -82,6 +82,11 @@ inline intptr_t* frame::link() const {
   return NULL;
 }
 
+inline bool frame::can_access_link(Thread *t) const {
```

Comment:
Did you test zero? Would this not just crash it?

Reply:
I did not yet test it, I'm waiting for our CI. But you're correct, I look into this.